client-side(frontend)/lifebird/src/Components1/login.js

```diff
@@ -0,0 +1,43 @@
+import react from 'react';
```

Comment:
Format the document with the prettier - code - formatter from the vscode. please add this as extension in your vscode